jwql/shared_tasks/run_pipeline.py

```diff
@@ -157,51 +207,73 @@ def run_save_jump(input_file, short_name, work_directory, instrument, ramp_fit=T
         # and use the run() method so that we can set parameters
         # progammatically.
         model = Detector1Pipeline()
+        params = {}
 
         # Always true
         if instrument == 'nircam':
-            model.refpix.odd_even_rows = False
+            params['refpix'] = dict(odd_even_rows=False)
 
         # Default CR rejection threshold is too low
-        model.jump.rejection_threshold = 15
+        #params['jump'] = dict(rejection_threshold=15)
```

Comment:
I'm unsure about this one. The default value for the parameter is 3. If you talk to some people, they say that mathmatically, 3 should be correct. On the other hand, I've seen a few help desk calls where people say the jump step is flagging too many pixels as jumps. That it's flagging noise rather than real jumps. And I assume that most users don't even check their data to see what was flagged.

Reply:
I know that the cosmic ray monitor used this version of the pipeline. I think the bad pixel monitor is the other one that uses it. Whether it's the right change to make depends on the goal of the monitor, and I don't actually know *why* the bad pixel monitor was doing this, in particular.

Reply:
For the bad pixel monitor, I think it was to avoid over-flagging as well. Given that this PR makes it possible for instrument monitors to pass parameter values to the pipeline calls, maybe it makes more sense to remove this parameter here, and have each monitor pass whatever parameters are appropriate? And the CR monitor is coming back at some point, but in a very different format that doesn't involve any pipeline calls.

Reply:
That's a good question. I guess the question is whether the save_jump pipeline is asking for different things, or just asking for custom parameters. I think, at the time, that it was using `call()` when the standard pipeline function in JWQL was using `run()` made me reluctant to try to combine together, but if we're at a point where we can just have one pipeline function, it will make the code path a lot easier (I think).

Reply:
It does seem like the only difference is the custom parameters. Using `run()` is frowned upon because parameters that are not explicitly set by the user fall back to default values that are not optimal. With `call()`, unspecified parameters are grabbed from parameter reference files, which contain better values.

Reply:
Fair. I expect that the functions in the JWST pipeline_tools.py file may have been written before all of that was specified.

Reply:
To me, this seems like a good change to make, but probably not in this PR. I feel as if it would add another big test burden, because we'd also have to start testing a bunch of files with the bad pixel monitor to make sure things are running the same. Assuming you approve, my inclination is to file an issue to combine these functions into a single function, and then resolve the conversation here. Then, once we've merged this in, I can work that issue.